cartography/models/aws/inspector/findings.py

```diff
@@ -0,0 +1,123 @@
+from dataclasses import dataclass
+
+from cartography.models.core.common import PropertyRef
+from cartography.models.core.nodes import CartographyNodeProperties
+from cartography.models.core.nodes import CartographyNodeSchema
+from cartography.models.core.nodes import ExtraNodeLabels
+from cartography.models.core.relationships import CartographyRelProperties
+from cartography.models.core.relationships import CartographyRelSchema
+from cartography.models.core.relationships import LinkDirection
+from cartography.models.core.relationships import make_target_node_matcher
+from cartography.models.core.relationships import OtherRelationships
+from cartography.models.core.relationships import TargetNodeMatcher
+
+
+@dataclass(frozen=True)
+class AWSInspectorNodeProperties(CartographyNodeProperties):
+    id: PropertyRef = PropertyRef('id')
+    arn: PropertyRef = PropertyRef('arn')
+    awsaccount: PropertyRef = PropertyRef('awsaccount')
+    name: PropertyRef = PropertyRef('title')
+    instanceid: PropertyRef = PropertyRef('instanceid')
+    ecrimage: PropertyRef = PropertyRef('ecrimageid')
+    ecrrepositoryid: PropertyRef = PropertyRef('ecrrepositoryid')
+    severity: PropertyRef = PropertyRef('severity')
+    firstobservedat: PropertyRef = PropertyRef('firstobservedat')
+    updatedat: PropertyRef = PropertyRef('updatedat')
+    description: PropertyRef = PropertyRef('description')
+    type: PropertyRef = PropertyRef('type')
+    cvssscore: PropertyRef = PropertyRef('cvssscore')
+    protocol: PropertyRef = PropertyRef('protocol')
+    portrange: PropertyRef = PropertyRef('portrange')
+    portrangebegin: PropertyRef = PropertyRef('portrangeend')
+    vulnerabilityid: PropertyRef = PropertyRef('vulnerabilityid')
+    referenceurls: PropertyRef = PropertyRef('referenceurls')
+    relatedvulnerabilities: PropertyRef = PropertyRef('relatedvulnerabilities')
+    source: PropertyRef = PropertyRef('source')
+    sourceurl: PropertyRef = PropertyRef('sourceurl')
+    status: PropertyRef = PropertyRef('status')
+    vendorcreatedat: PropertyRef = PropertyRef('vendorcreatedat')
+    vendorseverity: PropertyRef = PropertyRef('vendorseverity')
+    vendorupdatedat: PropertyRef = PropertyRef('vendorupdatedat')
+    vulnerablepackageids: PropertyRef = PropertyRef('vulnerablepackageids')
+    region: PropertyRef = PropertyRef('Region', set_in_kwargs=True)
+    lastupdated: PropertyRef = PropertyRef('lastupdated', set_in_kwargs=True)
```

Comment:
extra index for these props

1. vulnerabilityid
2. instanceid
3. ecrimageid
4. ecrrepositoryid
5. cvssscore

Reply:
Added for cvssscore. The others I imagine that they'd be queried on by their attached nodes e.g. to instances or the image nodes.